spec/components/tooltip.js

```diff
@@ -0,0 +1,152 @@
+import React, { PureComponent } from 'react';
```

Comment:
Can you make a story for it (and remove spec) so we can test it?

Reply:
Done! 🎉